coupon.go

```diff
@@ -15,15 +15,15 @@ const (
 // For more details see https://stripe.com/docs/api#create_coupon.
 type CouponParams struct {
 	Params           `form:"*"`
-	AmountOff        *int64  `form:"amount_off"`
-	Currency         *string `form:"currency"`
-	Duration         *string `form:"duration"`
-	DurationInMonths *int64  `form:"duration_in_months"`
-	ID               *string `form:"id"`
-	MaxRedemptions   *int64  `form:"max_redemptions"`
-	Name             *string `form:"name"`
-	PercentOff       *int64  `form:"percent_off"`
-	RedeemBy         *int64  `form:"redeem_by"`
+	AmountOff        *float64 `form:"amount_off"`
```

Comment:
Was changing `AmountOff` and not `PercentOff` in parameters intentional? It kind of looks like they should both be the same (and I would've thought they'd be floats themselves, but integers seem to still be recommended in the docs).

ptal @remi-stripe

Reply:
@brandur-stripe Sorry, I changed the wrong one I think. Only `percent_off` should be a float. It's for the latest API version we did: https://stripe.com/docs/upgrades#2018-07-27

> The percent_off field of coupons was changed from Integer to Float, with a precision of two decimal places.